src/ui/public/styles/sidebar.less

```diff
@@ -76,6 +76,10 @@
       color: @sidebar-header-color;
       border: 1px solid;
       border-color: @sidebar-header-border;
+
+      h3 {
+        font-size: 14px;
+      }
```

Comment:
Instead of styling this element, could you create a class called `sidebar-list-header-label` with these styles, and apply it to the `h3` in the markup? This makes the styles a little less tightly coupled to the markup.